packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -52,119 +62,251 @@ class _DeepLinkPageState extends State<DeepLinkPage>
 
   @override
   Widget build(BuildContext context) {
-    return ListView(
-      children: [
-        Row(
-          children: [
-            Expanded(
-              child: Text(
-                'All deep links',
-                style: Theme.of(context).textTheme.bodyLarge,
+    return DefaultTabController(
+      length: 3,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            children: [
+              Expanded(
+                child: Text(
+                  'All deep links',
+                  style: Theme.of(context).textTheme.bodyLarge,
+                ),
               ),
-            ),
-            InkWell(
-              onTap: () {
-                controller.bundleByDomain = !controller.bundleByDomain;
-              },
-              child: const Text('Bundle by domain'),
-            ),
-            const SizedBox(width: denseSpacing),
-            SearchBar(
-              leading: const Icon(Icons.search),
-              onChanged: (value) {
-                controller.searchContent = value;
-              },
-              constraints: BoxConstraints.tight(
-                Size(defaultSearchFieldWidth, defaultTextFieldHeight),
+              const SizedBox(width: denseSpacing),
+              SearchBar(
```

Comment:
Are you able to use `SearchField` from `search.dart`? There are a few examples of this throughout DevTools, but this would give you a search field styled consistently with the rest of DevTools that supports navigation through search results.

Reply:
is  SearchField paired with  SearchableFlatTable?

Reply:
They are not tightly coupled no. They can be used together (like for the CPU profiler Method Table, for example), but you can also use SearchField independently. It just requires a search controller, so you'l make your deep links controller mixin SearchControllerMixin for that. If you look at usages of `SearchField` in DevTools, you'll see several examples of using the `SearchField` widget with a controller that mixes in `SearchControllerMixin`. The logging screen and logging controller are an example of this.